aiida/restapi/translator/node.py

```diff
@@ -668,5 +673,16 @@ def get_node_shape(ntype):
                     "label": linktype
                 })
 
+        # count total no of nodes
+        qb = QueryBuilder()
+        qb.append(Node, tag="main", project=['id'], filters=self._id_filter)
+        qb.append(Node, tag="in", project=['id'], input_of='main')
+        no_of_inputs = qb.count()
+
+        qb = QueryBuilder()
+        qb.append(Node, tag="main", project=['id'], filters=self._id_filter)
+        qb.append(Node, tag="out", project=['id'], output_of='main')
+        no_of_outputs = qb.count()
+
 
-        return {"nodes": nodes, "edges": edges}
+        return {"nodes": nodes, "edges": edges, "total_no_of_incomings": no_of_inputs, "total_no_of_outgoings": no_of_outputs}
```

Comment:
Is the `total_` prefix needed?
What about `no_of_incomings`?

Reply:
I think it's more common to use `num_` instead of `no_`